overlord/overlord.go

```diff
@@ -105,25 +105,26 @@ func New() (*Overlord, error) {
 	}
 	o.addManager(hookMgr)
 
-	snapMgr, err := snapstate.Manager(s)
```

Comment:
are the changes here needed? as far as I can tell snapstate.Manager itself doesn't do anything interesting, and everything is done in Ensure which happens after New is done

Reply:
You are right, I reverted this change.